.eslintrc

```diff
@@ -1,15 +1,190 @@
 {
-  "parser": "babel-eslint",
-  "extends": "airbnb/base",
```

Comment:
Why remove airbnb? Just wondering! It has excellent style guides and path recognition that can help during dev time.

Also, airbnb can teach Turing students quite a bit about JavaScript when they read why a rule is a rule 😄